schema_based_on_diagram.sql

```diff
@@ -0,0 +1,57 @@
+-- Create patients table
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE
+);
+
+-- Create medical_histories table
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(255)
+);
+
+CREATE INDEX ON medical_histories(patient_id)
+
+-- Create treatments table
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    code VARCHAR(255),
```

Comment:
- [ ] [OPTIONAL] Looking at the diagram I think this column's name should be type :+1: